src/components/DropdownMenu/DropdownMenu.vue

```diff
@@ -2,12 +2,14 @@
     <ul class="dropdown-menu">
         <slot name="prepend" />
 
-        <DropdownMenuItem
-            v-for="(item, index) of items"
-            :key="`item-${index}`"
-            v-bind="item"
-            @click="clickItem(item)"
-        />
+        <slot name="items">
+            <DropdownMenuItem
+                v-for="(item, index) of items"
+                :key="`item-${index}`"
```

Comment:
We could even prevent the parent having to do the loop.
```suggestion
        <slot name="items">
            <template v-for="(item, index) of items">
                <slot name="item" :item="item">
```